src/languages/es.ts

```diff
@@ -1756,6 +1756,7 @@ export default {
             reimburse: 'Reembolsos',
             categories: 'Categorías',
             tags: 'Etiquetas',
+            taxes: 'Impuestos',
```

Comment:
Is it verified?

Reply:
Waiting for confirmation

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1709801973344319

Reply:
Looks like we got our answer here! :) https://expensify.slack.com/archives/C01GTK53T8Q/p1709822387687259?thread_ts=1709801973.344319&cid=C01GTK53T8Q